plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java

```diff
@@ -52,6 +54,32 @@ public void setUp()
         assertQuery("SELECT count(*) FROM base_table2", "VALUES 3");
     }
 
+    @Test
+    public void testShowCreate()
+    {
+        assertUpdate("CREATE MATERIALIZED VIEW materialized_view_with_property " +
+                "WITH (partitioning = ARRAY['_date']) AS " +
+                "SELECT _bigint, _date FROM base_table1");
+        assertQuery("SELECT COUNT(*) FROM materialized_view_with_property", "VALUES 6");
```

Comment:
isn't that tested somewhere else?

Reply:
All the other tests in this class are disabled due to https://github.com/trinodb/trino/issues/5892, until that's fixed it's useful to test a bit extra here.

Reply:
The tests are disabled due to an unrelated issue. In any case the fix for that issue and re-enabling of tests is here: https://github.com/trinodb/trino/pull/5942/commits. Would be great to get that PR in .. another person in the community reached out to me saying they were hitting that issue as well. @sopel39 @raunaqmorarka